config/field.go

```diff
@@ -15,12 +15,23 @@ const (
 	FieldTemplate       = "template"
 )
 
+// FieldsDesc is a record of all available config fields and their description.
+var FieldsDesc = map[string]string{
```

Comment:
Suggestion: even with the documentation, I feel like `Desc` as too much meaning as `descending` (`asc` vs `desc`).

`FieldsDefinition`, `FieldsUsage` ?